gap/oper.gd

```diff
@@ -98,6 +98,8 @@ DeclareOperation("PartialOrderDigraphMeetOfVertices",
 [IsDigraph, IsPosInt, IsPosInt]);
 DeclareOperation("DigraphClosure", [IsDigraph, IsPosInt]);
 
+DeclareOperation("HamiltonianPath", [IsDigraph]);
```

Comment:
This should be declared as an attribute not an operation, so that it is not recalculated every time this is called. It should therefore go in the `attr.g*` files.